.github/workflows/gcp_deploy_prod.yaml

```
@@ -3,7 +3,7 @@ name: Deploy to GCS Prod
 on:
   push:
     tags:
-      - "20[0-9][0-9].[0-9][0-9].[0-9][0-9](-[0-9]+)?"
+      - "202[0-9].[0-9][0-9].[0-9][0-9]*"
```

Maybe simplify to allow any tag that starts with `20`. We don't need to be very precise here, I think.

```suggestion
      - "20*"
```

---

Long as the same tag is being pushed and no one pushes anything not following the pattern